src/Features/OnboardingSetUpShipping.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Onboarding - set up shipping.
+ *
+ * @package Woocommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Features;
+
+use \Automattic\WooCommerce\Admin\PluginsHelper;
+use \Automattic\WooCommerce\Admin\Notes\WC_Admin_Notes_Review_Shipping_Settings;
+
+/**
+ * This contains logic for setting up shipping when the profiler completes.
+ */
+class OnboardingSetUpShipping {
```

Comment:
@becdetat what do you think about adding some test coverage for this new class?

Reply:
@timmyc I did consider that, but IMO this class doesn't actually contain a lot of logic and the tests would largely be testing truisms - such as "does `explode( ':', $default_country )[0]` return the first part of `$default_country`".

Reply:
I do agree that a little test coverage around `set_up_free_local_shipping` might be useful just to check if the corect zones were added, but if you think it's not very useful, Bec, then we can skip.

Reply:
Ok I've bowed under the pressure and added a couple of unit tests :-D in 752ed29